ml-agents/mlagents/trainers/learn.py

```diff
@@ -73,6 +76,21 @@ def run_training(sub_id: int, run_seed: int, run_options, process_queue):
             docker_target_name=docker_target_name
         )
 
+    sampler = None
```

Comment:
nit: I think `sampler_config` is a better name for this. `sampler` implies it's an instance of a `Sampler`